raw-svo.valid.main.lua

```diff
@@ -4544,12 +4544,23 @@ function valid.devastate_arms_cripple()
 end
 
 -- happens on wrist fracture levels 4,5
+-- edit: This ability can also mutilate a mangled limb.
 function valid.devastate_arms_mangle()
   checkaction(dict.wristfractures.gone, true)
   lifevision.add(actions.wristfractures_gone.p)
 
-  valid.simplemangledrightarm()
-  valid.simplemangledleftarm()
+  if affs.mangledrightarm
+  then
```

Comment:
we usually style if-then constructs like this:
~~~
if condition then
  doStuff()
else
  doOtherStuff()
end
~~~